playground/ProxylessEndToEnd/ProxylessEndToEnd.AppHost/Program.cs

```diff
@@ -0,0 +1,25 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+var builder = DistributedApplication.CreateBuilder(args);
+
+builder.AddProject<Projects.ProxylessEndToEnd_ApiService>("api")
+    .WithEndpoint("http", ea =>
+    {
+        ea.UriScheme = "http";
```

Comment:
Is this required?

Reply:
Yes. We do "magic" today in the other WithEndpoint overload where the name is also the scheme.